modules/post/linux/gather/enum_containers.rb

```diff
@@ -0,0 +1,94 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Linux Container Enumeration',
+      'Description' => %q{
+        This module attempts to enumerate containers running on the target machine.
+        Supports Docker, LXC and RKT.
+      },
+      'License' => MSF_LICENSE,
+      'Author' => ['Mat Rollings [stealthcopter]'],
```

Comment:
I don't think we support this format at the moment. If you'd like to have your handle in the code, you can add it in as a comment, but this Author field will be used by our backend systems, so most times people either set it to their personal name and then have a trailing comment afterwards with their handle, or they set it to their GitHub username.

Reply:
Thanks will update